prime-router/src/test/kotlin/azure/BatchDeciderTests.kt

```diff
@@ -104,17 +99,18 @@ class BatchDeciderTests {
         }
 
         // Invoke batch decider run
+        var result1 = BatchDeciderFunction(engine).determineQueueMessageCount(oneOrganization.receivers[0], null)
+        var result2 = BatchDeciderFunction(engine).determineQueueMessageCount(oneOrganization.receivers[0], null)
         BatchDeciderFunction(engine).run("", context = null)
```

Comment:
I don't follow how these runs change the result values for your test since they are run after.

Reply:
They do not change it - this is the test for when 'onlyOncePerDay' is set to false. It should generate the same result each time it is run (which it will. The test is silly since this is a mocked case, but if someone changed underlying logic this would catch it.